gradle/license/allowed-licenses.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
This is probably a feature for the plugin and not our build, but it would be great if this json could include the rules for license reproduction rather than us coding it ourselves. (IE. Apache needs x type license, MIT needs full repro, etc ...)